getnacelle/nacelle-js
# chore: update devDependencies

### WHY are these changes introduced?

#### On Paper

Addresses [LS-1336](https://nacelle.atlassian.net/browse/LS-1336)

> Update nacelle-js packages to use latest Vite



#### In Practice

[`@nacelle/vue@0.0.12`](https://www.npmjs.com/package/@nacelle/vue/v/0.0.12)'s `dist/nacelle-vue.es.js` looks as though it wasn't build + published correctly. This PR is an opportunity to redeploy our Vite-built packages.

### WHAT is this pull request doing?



- Upgrades `vite` and `@vitejs/plugin-vue` to their `@latest` versions
- Updates `.github/workflows/pr-health-checks.yml` to use a specific version of the workflow, with updated syntax

## 

### How to Test

- check out the `LS-1336-use-latest-vite` branch
- `npm run test:ci` from the monorepo root
  - all unit tests should pass
